### PR TITLE
Chapter 7 implementation

### PR DESCRIPTION
**Pull request by czegreanu**
Add details fragment

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (cosminstirbu) [inline app/.gitignore]: Hmm - in my experience we usually have a single .gitignore file in the root folder

File: app/build.gradle [hunk 2]
```diff
@@ -31,7 +41,10 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.1.0'
     implementation 'androidx.core:core-ktx:1.2.0'
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    testImplementation 'junit:junit:4.12'
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```
Comment (cosminstirbu) [inline app/build.gradle]: Is 'androidx.legacy:legacy-support-v4:1.0.0' actually needed :-?

File: app/build.gradle [hunk 3]
```diff
@@ -31,7 +41,10 @@ dependencies {
...
+    implementation 'androidx.fragment:fragment-ktx:1.2.3'
+    implementation "androidx.core:core-ktx:1.2.0"
```
Comment (cosminstirbu) [inline app/build.gradle]: This is already referenced

File: app/src/main/java/zegreanu/cristi/criminalintent/CrimeFragment.kt [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+        crime = Crime()
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        val v = inflater.inflate(R.layout.fragment_crime, container, false)
```
Comment (cosminstirbu) [inline app/src/main/java/zegreanu/cristi/criminalintent/CrimeFragment.kt]: Let's rename to rootView

File: app/src/main/res/layout/activity_crime.xml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (cosminstirbu) [inline app/src/main/res/layout/activity_crime.xml]: Hmm, let's see if we can use `FragmentContainerView` mentioned in the tech video from last week
https://www.youtube.com/watch?v=RS1IACnZLy4

Also please check https://proandroiddev.com/android-fragments-fragmentcontainerview-292f393f9ccf

**Review by cosminstirbu — CHANGES_REQUESTED** [review]

File: app/.gitignore (hunk 1 above)
Comment (czegreanu) [reply app/.gitignore]: Generated ones, like the other project but I agree with you.

**Review by cosminstirbu — APPROVED** [review]

**Pull request merged**